src/main/java/step2/FourOperationFactory.java

```diff
@@ -0,0 +1,32 @@
+package step2;
+
+import step2.FourOperation.*;
+
+public class FourOperationFactory implements OperationFactory{
+    private static final String PLUS = "+";
+    private static final String MINUS = "-";
+    private static final String DIVIDE = "/";
+    private static final String MULTIPLY = "*";
+
+    @Override
+    public Operation getOperation(String operation) {
+
+        if(operation.equals(PLUS)) {
+            return new PlusOperation();
+        }
+
+        if(operation.equals(MINUS)) {
+            return new MinusOperation();
+        }
+
+        if(operation.equals(MULTIPLY)) {
+            return new MultiplyOperation();
+        }
+
+        if(operation.equals(DIVIDE)) {
+            return new DivideOperation();
+        }
```

Comment:
팩토리를 통해 연산자를 반환해주도록 설계를 해주셨네요! 👍 👍 
다음과 같이 리팩토링이 되면 더욱 깔끔해질 것 같아요 😄 

```suggestion
public class FourOperationFactory implements OperationFactory {
    private static final String PLUS = "+";
    private static final String MINUS = "-";
    private static final String DIVIDE = "/";
    private static final String MULTIPLY = "*";
    private final Map<String, Operation> operators = new HashMap<>();
    
    {
        operators.put(PLUS, new PlusOperation());
        operators.put(MINUS, new MinusOperation());
        operators.put(DIVIDE, new DivideOperation());
        operators.put(MULTIPLY, new MultiplyOperation());
    }

    @Override
    public Operation getOperation(String operation) {
        if (operators.contains(operation)) {
            return operators.get(operation);
        }
```